retrieval/scrape.go

```diff
@@ -285,13 +285,15 @@ type targetScraper struct {
 }
 
 const acceptHeader = `application/vnd.google.protobuf;proto=io.prometheus.client.MetricFamily;encoding=delimited;q=0.7,text/plain;version=0.0.4;q=0.3,*/*;q=0.1`
+const userAgentHeader = `Prometheus-scraper/1.0`
```

Comment:
I believe they are generally versioned along behavior of the scraper (interval, connection handling, HTTP headers, ...) rather than the software itself.
Thinking about it, this might be relevant from a security perspective as it could expose to scraped targets which (potentially vulnerable) version of Prometheus is running. 
Of course obfuscation is no security implementation either... but still worth considering.

@brian-brazil